### DISCUSSION TITLE
Help understanding

### DISCUSSION THREAD

**Discussion by kangulp** [FaithBeam/Sims-1-Complete-Collection-Widescreen-Patcher#35] [Q&A]
Hello, I just got a 34 inch 3440x1440 monitor and I am trying to run sims at that 21:9 resolution. I came across this code but have no idea how to use it. Can someone help? Thank you in advance.

**Comment by FaithBeam** [comment]
@kangulp here's a more encompassing guide of my program: https://pleasantsims.com/sims-1-widescreen-windowed/

Instead of using 1920x1080 you would use whatever resolutions work for your monitor. Here is a wiki page to find out which one's should work for you: https://github.com/FaithBeam/Sims-1-Complete-Collection-Widescreen-Patcher/wiki/Selecting-Valid-Resolutions
